src/lib.rs

```diff
@@ -43,15 +46,19 @@ pub fn run(args: clap::ArgMatches, current_dir: &PathBuf) -> Result<Vec<Warning>
             None => continue,
         };
 
-        let file_with_lines = match FileEntry::from(relative_path) {
+        let (fe, strs) = match FileEntry::from(relative_path) {
             Some(f) => f,
             None => continue,
         };
 
-        let lines = get_line_entries(file_with_lines.0, file_with_lines.1);
+        let mut lines = get_line_entries(&fe, strs);
 
-        let result = checks::run(lines, &skip_checks);
+        let result = checks::run(&lines, &skip_checks);
         warnings.extend(result);
+
+        if autofix && fixes::run(&mut warnings, &mut lines) > 0 {
```

Comment:
It seems to me, by running corrections after the `warnings.extend(result)`, in fact we restart the check for already fixed ones (just unnecessary iterations)
Perhaps this is more correct to do on the `result`?

```suggestion
        if autofix && fixes::run(&mut result, &mut lines) > 0 {
```

Reply:
It seems you found a bug )
I will check.

Reply:
I fixed this and also added a test for fixing multiple files.